pkg/util/util.go

```diff
@@ -172,3 +177,28 @@ func Retry(operation retryFunc, retryCount int, initialDelayMilliseconds int) er
 
 	return err
 }
+
+func Lgetxattr(path string, attr string) ([]byte, error) {
```

Comment:
It doesn't look like this function needs to be exported, can we rename to `lgetxattr` and add a comment?

I'd also love to have a test covering this behavior, since it's pretty inscrutable to me personally.

Reply:
Sure, I can add a comment there.

Regarding tests, I can take a look at how to write a test(either a unit test or e2e test). But if you have any pointers or suggestions, that'd be great.